src/h3/mod.rs

```diff
@@ -415,6 +417,8 @@ impl Config {
             max_header_list_size: None,
             qpack_max_table_capacity: None,
             qpack_blocked_streams: None,
+            dgram_poll_threshold: None,
```

Comment:
Instead of using `Option` here, we could just set the default value to `std::usize::MAX` I think, as it seems like it would simplify things as we wouldn't need to deal with `None` anymore later on.

Reply:
Also, type should probably be `u64` as this doesn't really refer to any "memory-related quantities".

Reply:
I was trying to be clever, let me try the KISS option.